pages/metadata/productblocks.tsx

```diff
@@ -1,7 +1,17 @@
-import React from 'react';
-
-import { WfoProductBlocksPage } from '@orchestrator-ui/orchestrator-ui-components';
-
-export const ProductBlocksPage = () => <WfoProductBlocksPage />;
-
-export default ProductBlocksPage;
+import {
+    PolicyResource,
+    WfoPolicyRenderPageFallback,
+    WfoProductBlocksPage,
+  } from '@orchestrator-ui/orchestrator-ui-components';
+  import React from 'react';
+  
+  export const ProductBlocksPage = () => {
+    return (
+      <WfoPolicyRenderPageFallback resource={PolicyResource.NAVIGATION_METADATA}>
```

Comment:
I think there should be a better way then adding the policy fallback on each page.

Reply:
searched it up, this could be useful: https://nextjs.org/docs/pages/building-your-application/authentication#protecting-routes-with-middleware

Reply:
Mwa.. we have been able to avoid middleware so far and it would be nice to keep that up

Reply:
So I will keep this as it is. is that ok with you guys?